app/assets/stylesheets/common/admin/customize_install_theme.scss

```diff
@@ -0,0 +1,112 @@
+.install-theme {
```

Comment:
For `scss` files we've been trying to prefer dashes instead of underscores in filenames. I know it's a little inconsistent now but most are dashes and we should probably stick to it.